### ISSUE TITLE
Controls as ‘Edit variation', 'change background', 'delete variation' are not accessible using keyboard.

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#111]
User Experience:

User rely on keyboard/Screen Reader will find difficulty to interact with the controls if controls are not accessible through keyboard as user won't be able to perform functionality over such control.

 

Note: User credentials should NOT be included in the bug.

 

Repro Steps:

1. Open Azure portal with valid credentials and create a API Management.
2. Click Developer Portal button to navigate to the portal.
3. Wait portal to load in design mode
4. Navigate to the Left nav and select "Styles" button.
5. Navigate to the various controls of the "Styles" window.

6. Verify whether all controls are accessible or not via keyboard.

 

Actual Result:

‘Edit variation', 'change background', 'delete variation' buttons present in 'font' section, 'typography' section, 'heading' section, 'color' section, 'shadows' section 'button' section, 'menu' section etc.  and all shadow as  ’Inset', 'light' and 'medium’ shadows are not accessible using keyboard, present on ‘Style’ window.

 
Expected Result:

All controls as ‘Edit variation', 'change background', 'delete variation' buttons present in 'font' section, 'typography' section, 'heading' section, 'color' section, 'shadows' section 'button' section, 'menu' section etc.  and all shadow as  ’Inset', 'light' and 'medium’ shadows should be accessible using keyboard, present on ‘Style’ window.

Note:

Similar issue exist with Chrome.

Similar issue exist for  all ‘Edit variation', 'change background', 'delete variation’ controls present in style page.

MAS Reference:

MAS 2.1.1 - Keyboard (36)

Reference Links

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries.
